engines/query-sparql-rdfjs-lite/webpack.config.ts

```diff
@@ -0,0 +1,10 @@
+import { createConfig } from '../../webpack.config';
```

Comment:
Given the `.ts` rename, are you sure our compiler will always compile this down to `.js`? (since it's not in `lib/`)